### PR TITLE
torrentday fix

### PR DESCRIPTION
**Pull request by serensoner**
### Description of what this fixes:
torrentday captcha problem
in order to solve this, user enters the cookie details into the login details of torrentday selection. if these details exist, for torrentday, the browser also sends the cookie details in request header.

### Related issues:
#5869 #6729 #6530 #6518 #6515 #6514 #6548 #6542 #6562 #6590 and many more

### CODE REVIEW

File: couchpotato/core/media/_base/providers/base.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ def login(self):
                 self.login_failures = 0
                 return True
 
-            error = 'unknown'
+            error = 'unknowntest'
```
Comment (RuudBurger) [inline couchpotato/core/media/_base/providers/base.py]: Please don't change the base one. This error is used on other places.

File: couchpotato/core/plugins/base.py [hunk 2]
```diff
@@ -171,6 +172,10 @@ def urlopen(self, url, timeout = 30, data = None, headers = None, files = None,
         headers['Connection'] = headers.get('Connection', 'keep-alive')
         headers['Cache-Control'] = headers.get('Cache-Control', 'max-age=0')
 
+        if self.getName() is "TorrentDay":
+		cookies = self.getCookies()
+		headers['Cookie'] = cookies
+
```
Comment (RuudBurger) [inline couchpotato/core/plugins/base.py]: You can send headers in the main torrentday function (using the urlopen function). Putting it in base isn't the correct way.

**Comment by RuudBurger** [comment]
Please checkout the comments. Fix those and I can merge it. Thanks for the PR!

**Comment by relaxedricky** [comment]
https://github.com/CouchPotato/CouchPotatoServer/issues/6753 - Am I better off using this branch over the current master one?

Also, when do we expect this to be merged?

**Comment by RuudBurger** [comment]
Merged manually

**Comment by MadKezza** [comment]
when will this fix enter master branch .... and how do i switch branches as i think I'm stuck on master with QNAP NAS

**Comment by shawnmstout** [comment]
getting this error now when i implemented these changes

01-08 16:24:59 ERROR [edia._base.providers.base] Failed to parsing TorrentDay: Traceback (most recent call last):
  File "C:\Users\***\AppData\Roaming\CouchPotato\application\CouchPotato-3.0.1.win32\couchpotato\core\media\_base\providers\base.py", line 84, in getJsonData
  File "json\__init__.pyc", line 338, in loads
  File "json\decoder.pyc", line 366, in decode
  File "json\decoder.pyc", line 384, in raw_decode
ValueError: No JSON object could be decoded

**Comment by shawnmstout** [comment]
looks like they took classic away on 1/1/2017, please take a look at this thread

https://www.torrentday.com/forums.php?t=2053468

**Pull request closed without merging**